src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt

```diff
@@ -84,7 +84,7 @@ class FindPaths: CliktCommand(help = "Impute best path(s) using read mappings.")
         option("--path-keyfile", help = "Name of tab-delimited key file.  Columns for samplename and filename" +
                 " are required. Files must be either read mapping files (ending in _readMapping.txt) or fastq files. " +
                 "If using paired end fastqs, a filename2 column can be included. A value must be entered for " +
-                "either --key-file or --read-files.")
+                "either --path-keyfile or --read-files.")
```

Comment:
There seems to be a few references to --key-file in imputation.md?

Reply:
--key-file is used by MapReads, while --path-key-file is used by FindPaths. I did find one place that needed to be changed